### ISSUE TITLE
Element going outside of the window

### ISSUE THREAD

**Issue by karthikeyan676** [yjose/reactjs-popup#238]
Hi, this feature is not working. 

4. check if calculated position is going out of bounds of wrapper box or not. If yes repeat 1-3 for next position enum. By default wrapper box is window element

![image](https://user-images.githubusercontent.com/24937683/107154663-392c0b80-699a-11eb-8797-46212fcf580a.png)

**Comment by huksley** [comment]
Yeah, I have the same problem. Is there a possible fix for this? I was trying to change property`position={}` but nothing helps.

<img width="441" alt="image" src="https://user-images.githubusercontent.com/1594349/205303513-babae37a-c236-43b0-af28-3e31b9dfbab7.png">

Zooming out helps 

<img width="367" alt="image" src="https://user-images.githubusercontent.com/1594349/205303820-65b4233d-7505-4025-b486-9c3c78d018e0.png">
